neps/nep-0417.md

```diff
@@ -0,0 +1,202 @@
+---
+NEP: 417
+Title: Meta Transactions
+Author: Illia Polosukhin <ilblackdragon@gmail.com>, Egor Uleyskiy (egor.ulieiskii@gmail.com), Alexander Fadeev (fadeevab.com@gmail.com)
+DiscussionsTo: https://github.com/nearprotocol/neps/pull/417
+Status: Draft
+Type: Protocol Track
+Category: Runtime
+Created: 19-Oct-2022
+---
+
+## Summary
+
+In-protocol meta transactions allowing for third-party account to initiate and pay transaction fees on behalf of the account.
```

Comment:
nit: I wonder if this feature should be called delegate transactions instead of meta transactions.  I feel like that is a more precise name.